### DISCUSSION TITLE
Is it possible to control how much vertical space is left between the crop selection region and the inactive area?

### DISCUSSION THREAD

**Discussion by KevinJHall-Launchpad** [ValentinH/react-easy-crop#530] [Q&A] [answered]
We want to only have something like this where the inactive space (grey area) above and below is the same regardless of the image uploaded which will mean to grow and shrink the vertical height of the container as necessary.
![image](https://github.com/ValentinH/react-easy-crop/assets/145647170/efe6d545-2477-4de7-ba55-1059a87d7665)

Currently some images with more extreme widths and heights end up with more of this inactive space so we want to equalize this if possible.
![image](https://github.com/ValentinH/react-easy-crop/assets/145647170/f8ad46be-d7cf-417b-9182-0c28af0198bd)

**Comment by ValentinH** [comment]
What do you expect to do to "equalize"? I don't really understand what result you are looking for.

**Comment by KevinJHall-Launchpad** [comment]
Equalize was a bad word choice.

Sorry, I meant I want to minimize the grey area above and below the image even if it means making that section smaller.
![image](https://github.com/ValentinH/react-easy-crop/assets/145647170/f8ad46be-d7cf-417b-9182-0c28af0198bd)

To something like this
![image](https://github.com/ValentinH/react-easy-crop/assets/145647170/07af7947-3158-4a54-93b7-ed924f04a6fd)

Ideally, I would be able to achieve this with any input file, but I'm unsure of a configuration or approach to achieve this.

**Reply by ValentinH** [reply, marked as answer]
You need to compute the aspect ratio of the image selected by the user and then modify the Cropper container to have this ratio. Indeed, the cropper just fills its parent so it's the parent component responsibility to have the right dimensions.

If your user is selecting the image from disk, you can get inspiration from [this example](https://codesandbox.io/p/sandbox/busy-keldysh-y09komm059?file=%2Fsrc%2Findex.js)  to learn how to read the image.
Otherwise, you can use the `onMediaLoaded` prop to get the image dimensions once it's loaded to then compute the aspect ratio.
